sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,284 @@
+          'weather_location_selected',
+          -- wallpaper
+          'wallpaper_click',
+          'wallpaper_category_click',
+          'wallpaper_highlight_cta_click',
+          'wallpaper_highlight_dismissed',
+          -- topic_selection
+          'topic_selection_open',
+          'topic_selection_dismiss',
+          'topic_selection_topics_saved',
+          -- section
+          'sections_block_section',
+          'sections_follow_section',
+          'sections_unblock_section',
+          'sections_unfollow_section',
+          'inline_selection_click'
+        )
+      )
+    )
+)
+SELECT
+  submission_date,
+  client_id,
+  mozfun.map.get_key(event_details, 'newtab_visit_id') AS newtab_visit_id,
+  -- ANY_VALUE: because we do not expect those fields to differ within a visit
+  ANY_VALUE(app_version) AS app_version,
+  ANY_VALUE(os) AS os,
+  ANY_VALUE(channel) AS channel,
+  ANY_VALUE(locale) AS locale,
+  ANY_VALUE(country) AS country,
+  ANY_VALUE(homepage_category) AS homepage_category,
+  ANY_VALUE(newtab_category) AS newtab_category,
+  ANY_VALUE(organic_content_enabled) AS organic_content_enabled,
+  ANY_VALUE(sponsored_content_enabled) AS sponsored_content_enabled,
+  ANY_VALUE(sponsored_topsites_enabled) AS sponsored_topsites_enabled,
+  ANY_VALUE(organic_topsites_enabled) AS organic_topsites_enabled,
+  ANY_VALUE(newtab_search_enabled) AS newtab_search_enabled,
+  LOGICAL_OR(is_default_ui) AS is_default_ui,
+  LOGICAL_OR(event_category = 'newtab' AND event_name = 'opened') AS is_newtab_opened,
+  LOGICAL_OR(event_category = 'newtab.search' AND event_name = 'issued')
+  AND LOGICAL_OR(is_default_ui) AS is_search_issued,
+  LOGICAL_OR(event_category = 'newtab.search.ad' AND event_name = 'click')
+  AND LOGICAL_OR(is_default_ui) AS is_search_ad_click,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click', 'dismiss', 'thumb_voting_interaction', 'thumb_voting_interaction')
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_content_interaction,
+  LOGICAL_OR(event_category = 'pocket' AND event_name IN ('click'))
+  AND LOGICAL_OR(is_default_ui) AS is_content_click,
+  LOGICAL_OR(event_category = 'pocket' AND event_name IN ('impression'))
+  AND LOGICAL_OR(is_default_ui) AS is_content_impression,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click', 'dismiss')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_interaction,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_click,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('impression')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_impression,
```

Comment:
Nit: For this table, I'd prefer if we renamed `is_*` to `any_*` or `has_*`?  The latter seem more appropriate to an aggregate

Reply:
I am ok with any of those prefixes. I would prefer to go with a prefix that is commonly used across mozilla. Also would like @florezita and Ilana to weigh in on these matters

Reply:
I do not think there is a strong preference to change the prefix here.

Also, I am of the opinion that such matters (field names and metrics definitions) should be decided much earlier on in the requirement definition process.

What I think would help in subsequent releases, is a sign-off process for the requirement definitions (similar to an approval process used for PRs) from all parties involved.
This will hopefully avoid scope creep and ensure a better streamlined process for completing projects.

cc: @lucia-vargas-a @florezita

Reply:
All fair, the important thing is the description in any case